services/tripService.js

```diff
@@ -0,0 +1,42 @@
+import { validateId, validateTripName } from "../helpers/validate";
+import dbConnect from "../lib/dbConnect";
+import Trip from "../models/Trip";
+
+export async function getAllTrips() {
+  await dbConnect();
+  const trips = await Trip.find();
+  if (!Array.isArray(trips)) return { status: 500, error: "Internal server error" };
```

Comment:
What does this line do?

Reply:
Gets all trips from the DB. If theres not an array as a result, there might be a server error.

Reply:
Ah ok, thanks. I did not know this syntax yet.